src/tests/i5_3337u.rs

```diff
@@ -273,7 +273,7 @@ fn extended_features() {
             | ExtendedFeaturesEbx::CLFLUSHOPT
             | ExtendedFeaturesEbx::PROCESSOR_TRACE,
         ecx: ExtendedFeaturesEcx::from_bits_truncate(0),
-        _edx: 201326592,
+        edx: ExtendedFeaturesEdx::from_bits_truncate(201326592),
```

Comment:
Can we add the asserts for this too in the code below?

Reply:
This has only two most significant bits set. So, all the features are off except for the two.
So, I will add assertions that features are not available.